sass/tji/_data.scss

```diff
@@ -0,0 +1,54 @@
+/*--------------------------------------------------------------
+>>> EXPLORE THE DATA TABLE OF CONTENTS:
+----------------------------------------------------------------
+# GENERAL
+# CHARTS
+
+/*--------------------------------------------------------------
+## CHARTS
+--------------------------------------------------------------*/
+
+#js-TJIChartView {
+  width: 100%;
```

Comment:
Adding a width and margin-left here is fighting-with/overwriting how the flexboxgrid .row class is being used -- and is causing the chartview wrapper to expand past the content_area/primary container. This approach is brittle and adds complexity to our layout that is difficult. Could we find another way to achieve desired spacing? Do we need these two styles?